js_modules/dagster-ui/packages/ui-core/src/assets/AssetEventDetail.tsx

```diff
@@ -8,124 +9,87 @@ import {RunlessEventTag} from './RunlessEventTag';
+        return {
+          icon: <Icon name="observation" color={Colors.accentGreen()} size={16} />,
+          label: 'Observed',
+        };
+      case 'FailedToMaterializeEvent':
+        return {
+          icon: <Icon name="run_failed" color={Colors.accentRed()} size={16} />,
+          label: 'Failed',
+        };
+    }
+  }, [event.__typename]);
+
   return (
     <Box padding={{horizontal: 24, bottom: 24}} style={{flex: 1}}>
-      <Box padding={{vertical: 24}} border="bottom" flex={{alignItems: 'center', gap: 12}}>
-        <Heading>
-          <Timestamp timestamp={{ms: Number(event.timestamp)}} />
-        </Heading>
-        {isRunlessEvent(event) ? <RunlessEventTag tags={event.tags} /> : undefined}
-      </Box>
-      <Box
-        style={{display: 'grid', gridTemplateColumns: '1fr 1fr 1fr 1fr', gap: 16}}
-        border="bottom"
-        padding={{vertical: 16}}
-      >
-        <Box flex={{gap: 4, direction: 'column'}}>
-          <Subheading>Event</Subheading>
-          {event.__typename === 'MaterializationEvent' ? (
-            <Box flex={{gap: 4}}>
-              <Icon name="materialization" />
-              Materialization
-            </Box>
+      <Box padding={{vertical: 24}} border="bottom" flex={{direction: 'column', gap: 8}}>
+        <Box flex={{direction: 'row', gap: 12, alignItems: 'center'}}>
+          <Heading>
+            <Timestamp timestamp={{ms: Number(event.timestamp)}} />
+          </Heading>
+          {isRunlessEvent(event) ? <RunlessEventTag tags={event.tags} /> : undefined}
+        </Box>
+        <Box flex={{direction: 'row', gap: 4, alignItems: 'center'}}>
+          {icon}
+          {event.partition ? (
+            <>
+              Partition
+              <Link
+                to={assetDetailsPathForKey(assetKey, {
+                  view: 'partitions',
+                  partition: event.partition,
+                })}
+              >
+                {event.partition}
+              </Link>
+              {label.toLowerCase()}
+            </>
           ) : (
-            <Box flex={{gap: 4}}>
-              <Icon name="observation" />
-              Observation
-            </Box>
+            <span>{label}</span>
           )}
-        </Box>
-        {event.partition && (
-          <Box flex={{gap: 4, direction: 'column'}}>
-            <Subheading>Partition</Subheading>
-            {hidePartitionLinks ? (
-              event.partition
-            ) : (
-              <>
-                <Link
-                  to={assetDetailsPathForKey(assetKey, {
-                    view: 'partitions',
-                    partition: event.partition,
-                  })}
-                >
-                  {event.partition}
-                </Link>
-                <Link
-                  to={assetDetailsPathForKey(assetKey, {
-                    view: 'partitions',
-                    partition: event.partition,
-                    showAllEvents: true,
```

Comment:
Fwiw I think it's nice having a deep link to the all-events view on this page, but we could remove it and see if anyone is lost. I think we added this in response to user confusion over where the historical events for a single partition were

Maybe if this page supports filtering now it could offer to filter to the partition for you?

Reply:
cc @braunjj this is missing in the latest designs! Maybe we should add it back somewhere?